casual/casual-inbound-handler-api/src/test/groovy/se/laz/casual/jca/inbound/handler/buffer/PassThroughBufferHandlerTest.groovy

```diff
@@ -18,10 +20,22 @@ class PassThroughBufferHandlerTest extends Specification
     @Shared Method echoRequest = TestService.getMethod( "echo", InboundRequest.class )
     @Shared Method echoBuffer = TestService.getMethod( "echo", FieldedTypeBuffer.class )
     @Shared Method echoString = TestService.getMethod( "echo", String.class )
+    @Shared TestService proxyService
+    @Shared Proxy jndiObject
 
     def setup()
     {
         instance = new PassThroughBufferHandler()
+        proxyService = Mock( TestService )
+
+        Class[] c = new Class[1]
+        c[0] = TestService.class
+
+        jndiObject = (Proxy)Proxy.newProxyInstance(
```

Comment:
not needed - just replace with `Mock(Proxy)`, similarly the ForwardingInvocationHandler class is then not needed.